packages/frontend/src/components/accounts/ledger/SetupLedger.js

```diff
@@ -51,6 +50,10 @@ const SetupLedger = (props) => {
     const fundingOptions = parseFundingOptions(props.location.search);
     const shouldRenderRecaptcha = !fundingOptions && RECAPTCHA_CHALLENGE_API_KEY && isNewAccount && !ENABLE_IDENTITY_VERIFIED_ACCOUNT;
 
+    const has2fa = useSelector(selectAccountHas2fa);
+    // disable the Continue button if a user has 2fa enabled, or we don't know yet if it's disabled/enabled
+    const disable2fa = has2fa || typeof has2fa === 'undefined';
```

Comment:
nit: can we please name the selector `accountHas2fa` and change `disable2fa` --->  `accountMightHave2fa`

Reply:
done